src/components/formControlSection/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC, PropsWithChildren } from 'react';
+import {
+  Box,
+  FormControl as ChakraFormControl,
+  Flex,
+  FormErrorMessage,
+} from '@chakra-ui/react';
+
+import Tooltip from '../tooltip';
+import Text from '../text';
+
+import { InformationIcon } from '../icons';
+
+export type Props = {
+  id: string;
+  errorMessage?: string;
+  label?: string;
```

Comment:
I believe the label is always required, no?

Reply:
Otherwise you need to pass null in the ternary condition to do not showing the element in the DOM